tests/test_plan/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from django.db.migrations import Migration
+
+
+@pytest.fixture()
+def plan(mocker):
```

Comment:
Why do we need a `mocker` here?

Reply:
leftover from some "first attempts" code, thanks for catching!